lua/lualine/components/grapple.lua

```diff
@@ -32,9 +32,21 @@ function Component:update_status()
 
     local current = grapple.find({ buffer = 0 })
 
+    local app_ok, app = pcall(require, "grapple.app")
+    local quick_select = nil
+    if app_ok then
+        quick_select = app.get().settings:quick_select()
+    end
+
```

Comment:
No need for the pcall here. At this point we can guarantee that grapple is available.

```suggestion
    local App = require("grapple.app")
    local app = App.get()
    local quick_select = app.settings:quick_select()
```